Source/WebCore/rendering/updating/RenderTreeUpdater.cpp

```diff
@@ -308,6 +308,15 @@ void RenderTreeUpdater::updateSVGRenderer(Element& element)
 {
     ASSERT(element.needsSVGRendererUpdate());
     element.setNeedsSVGRendererUpdate(false);
+
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+    if (element.document().settings().layerBasedSVGEngineEnabled()) {
+        if (auto* renderer = element.renderer())
+            renderer->setNeedsLayout();
+        return;
+    }
+#endif
```

Comment:
maybe integrate this into the if (element.renderer()) section below.

Reply:
Done.